src/vs/workbench/contrib/debug/common/disassemblyViewInput.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { EditorInput } from 'vs/workbench/common/editor/editorInput';
+import { localize } from 'vs/nls';
+
+export class DisassemblyViewInput extends EditorInput {
+
+	static readonly ID = 'debug.disassemblyView.input';
+
+	override get typeId(): string {
+		return DisassemblyViewInput.ID;
+	}
+
+	static _instance: DisassemblyViewInput;
```

Comment:
I wonder why only a single Disassembly view is supported.
In a multi-threaded debugging scenario multiplexing multiple threads into a single viewer will result in an "interesting" user experience...

Reply:
ATM, it is to maintain parity with VS's disassembly view.  We can have a discussion about this feature for the next iteration.